### DISCUSSION TITLE
Stop not possible by button

### DISCUSSION THREAD

**Discussion by Rudiberto** [AlexandrePTJ/kemai#126] [General]
On Ubuntu 24.04 everything seems to work fine. I can see all available positions, but STOP is not possible by clicking on the button:

![grafik](https://github.com/user-attachments/assets/883343f2-c01d-40a8-b045-cc8bdf2380bd)

And this despite the fact that a "start again" on the task tab stops it and starts a new one! 

![grafik](https://github.com/user-attachments/assets/6f0ac5cf-7b6e-43fc-9f72-656bd86392a7)

**Comment by AlexandrePTJ** [comment]
I will look at this. I created issue #128
